### PR TITLE
[WIP] Kube-batch perf test

### PR DESCRIPTION
**Pull request by shivramsrivastava**
**What this PR does / why we need it**:
This PR adds the scheduler_perf test to kube-batch, this is similar to the scheduler_perf test found in the k8s project for the default scheduler.

**Special notes for your reviewer**:
1) This are the initial changes.

TODO:
- [ ] Adding the test framework api-server with CRD support. 
- [ ] Adding Pod-Affinity/Anti-Affinity and the Node-Affinity/Anti-Affinity tests.
- [ ] Updating the README
- [ ] Running the test and generate sample graphs
- [X] Added the initial code and updated vendor.
- [X] Added methods to run kube-batch from within the perf test.

### CODE REVIEW

File: test/integration/framework/etcd.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
+/*
```
Comment (k82cn) [inline test/integration/framework/etcd.go:1]: why do we copy code here? we should also use import for those package.

**Comment by k82cn** [comment]
@shivramsrivastava , are you still working on this?

**Comment by shivramsrivastava** [comment]
I am putting an hold on this task for now, since i am unable to resolve the api-server issue for crd support.
I will take up others issues in the meantime.
If you have any suggestions on which issue to take let me know, please?

**Comment by k82cn** [comment]
@TommyLike , @hzxuzhonghu , would you help to review this PR and get it done ASAP? we're testing the performance of kube-batch, and this is important for us.

**Pull request closed without merging**